### ISSUE TITLE
Couldn't load plugin after upgrade

### ISSUE THREAD

**Issue by KrOstir** [EarthObservation/rvt-qgis#9]
The upgrade issues the following error:
```
Couldn't load plugin 'rvt-qgis' due to an error when calling its classFactory() method
```

QGIS restart loads the plugin and everything works as intended.

![qgis-bin_210202_075412](https://user-images.githubusercontent.com/13023733/106565162-a79e4300-652e-11eb-85d9-a94b72b8019a.png)

Windows 10
Python version: 3.7.0
QGIS version: 3.16.2-Hannover

**Comment by zm8597** [comment]
I am aware of this problem. When you upgrade plugin to new version, QGIS reloads plugin code but doesn't reload library (rvt-py) code. Library code is only loded when you start QGIS, this is why you have to restart QGIS when upgrading to newer version. I will take a look and try to figure something out to solve this.

**Comment by zm8597** [comment]
Bug was resolved in plugin version 0.4.0, now plugin refreshes rvt-py library.
